src/fragments/lib-v1/predictions/js/getting-started.mdx

```diff
@@ -0,0 +1,163 @@
+> Prerequisite: [Install and configure the Amplify CLI](/cli/start/install)
+
+## Create new backend
+
+Run the following command in your project's root folder:
+
+```bash
+amplify add predictions
+```
+
+The CLI will prompt configuration options for the Predictions category such as what type of use case you have (identifying objects from an image, translating text, etc) and default or advanced settings.
+
+The Predictions category utilizes the Auth category behind the scenes to authorize your app to perform AI/ML actions.
+
+The `add` command automatically creates the backend configuration. Once all your configuration is complete run the following:
+
+```bash
+amplify push
+```
+
+A configuration file called `aws-exports.js` will be copied to your configured source directory, for example `./src`.
+
+## Configure the frontend
+
+import js0 from '/src/fragments/lib/common/js/import_configuration.mdx'
+
+<Fragments fragments={{ js: js0 }} />
+
+import rn1 from '/src/fragments/lib/common/react-native/import_configuration.mdx'
```

Comment:
These are easy to miss - but should they point at `lib-v1` fragments?

Reply:
Indeed they should, thanks!